README.md

```diff
@@ -80,12 +80,22 @@ See [reviewdog documentation for filter mode](https://github.com/reviewdog/revie
 
 ### `fail_on_error`
 
+**Deprecated**. This option is no longer recommended for use and will be removed in future versions.
+
 Optional. Exit code for reviewdog when errors are found [`true`, `false`].
 
 The default is `false`.
 
 See [reviewdog documentation for exit codes](https://github.com/reviewdog/reviewdog/tree/master#exit-codes) for details.
 
+### `fail_level`
+
+Optional. The level of failures that will cause the action to fail [any,info,warning,error].
+
+The default is `error`.
```

Comment:
Description hasn't been fixed yet.
It might be better to use the same sentence as in action-rubocop.

```suggestion
Optional. If set to none, always use exit code 0 for reviewdog. Otherwise, exit code 1 for reviewdog if it finds at least 1 issue with severity greater than or equal to the given level. Possible values: [none, any, info, warning, error] Default is none.

The default is `none`.
```
ref. https://github.com/reviewdog/action-rubocop?tab=readme-ov-file#inputs